README.md

```diff
@@ -26,27 +26,30 @@ The module handles the following account settings:
     - Restrict: traffic that originates outside the specified allowed IP addresses is blocked
 - Global shell settings (on - off)
 - Public access group (on - off)
-- Financial Services Validated products (on - off)
-- User List Visibility restrictions (on - off)
 
 The module supports creating and updating settings that are applied with the `terraform apply` command. With objects
 affected by the `destroy` command, the module preserves the most recent setting and doesn't change objects that are
 configured outside of Terraform's scope.
 
 ### Current limitations:
-The module currently supports setting the following FSCloud requirements using the RestAPI provider until the IBM provider supports these settings:
+The module currently does not support setting the following FSCloud requirements using the IBM provider:
 
 - Check whether user list visibility restrictions are configured in IAM settings for the account owner
 - Check whether the Financial Services Validated setting is enabled in account settings
 
-Tracking issue with IBM provider -> https://github.com/IBM-Cloud/terraform-provider-ibm/issues/4204
+Tracking issue with IBM provider -> <https://github.com/IBM-Cloud/terraform-provider-ibm/issues/4204>
+
+If you need to manage these FSCloud requirements via Terraform, please see [this submodule](./modules/restapi-options) which uses the RestAPI provider to manage these settings.
 
 <!-- Below content is automatically populated via pre-commit hook -->
 <!-- BEGIN OVERVIEW HOOK -->
 ## Overview
 * [terraform-ibm-iam-account-settings](#terraform-ibm-iam-account-settings)
+* [Submodules](./modules)
+    * [restapi-options](./modules/restapi-options)
```

Comment:
perhaps ask in the team channel for suggestions on the naming here. I was thinking something with the word "experimental" in it, but lets see what other think?